src/main/java/de/siegmar/logbackawslogsjsonencoder/AwsJsonLogEncoder.java

```diff
@@ -276,7 +282,13 @@ private PatternLayout buildPattern(final String pattern) {
     private void append(final SimpleJsonEncoder json, final String key,
                         final Map<String, Object> values) {
         if (!values.isEmpty()) {
-            json.appendToJSON(key, values);
+            if (key != null && key.length() > 0) {
+                json.appendToJSON(key, values);
+            } else {
+                values.forEach((k, v) -> {
+                    json.appendToJSON(k, v);
```

Comment:
As you described in the README, this would potentially overwrite existing attributes like timestamp, level, etc. That should not be possible as it could break the entire logging.

Reply:
An alternative solution could be an option like `onlyPrefixConflictingMdcKeys` (a better name could probably be found). That will compare each MDC key to a set of reserved words (timestamp, level, logger, thread, message, etc) and then only prefix when there is a conflict. That should be relatively simple but could be slightly confusing if you ever have a conflict.